migrations/1679042894877-oauth-provisioning-school-feature.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { alert, error } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+// use your own name for your model, otherwise other migrations may fail.
+// The third parameter is the actually relevent one for what collection to write to.
+const School = mongoose.model(
+	'school202303170948',
+	new mongoose.Schema(
+		{
+			features: [{ type: String }],
+		},
+		{
+			timestamps: true,
+		}
+	),
+	'schools'
+);
+
+// How to use more than one schema per collection on mongodb
+// https://stackoverflow.com/questions/14453864/use-more-than-one-schema-per-collection-on-mongodb
+
+module.exports = {
+	up: async function up() {
+		await connect();
+		// ////////////////////////////////////////////////////
+		// Make changes to the database here.
+		// - Only use models declared in the migration.
+		// - Make sure your migration is idempotent. It is not guaranteed to run only once!
+		// - Avoid any unnecessary references, including environment variables. If you have to run the migration on a single instance, use SC_THEME.
+
+		await School.updateMany(
```

Comment:
I do not like the idea to add this on every school on every instance. 
It should only added on nbc. Also the logic should be handle it. I think a test for it already exists.

Not the nice way, but in the past we use the SC_THEME env as filter for instance specific migrations.

Reply:
@Metauriel can you please look into it, maybe with @IgorCapCoder

Reply:
I dont think oauth provisioning is enabled on every school on every instance. Either it has the wrong name, or its applied the wrong way.